File: pinyougou-sellergoods-interface/src/main/java/com/pinyougou/sellergoods/service/BrandService.java
package com.pinyougou.sellergoods.service;

import java.util.List;
import java.util.Map;

import com.pinyougou.pojo.TbBrand;

import entity.PageResults;

/**
 * 品牌接口
 * @author DarryRing
 *
 */

public interface BrandService {
	
	List<TbBrand> findAll();
	
	PageResults findPaging(int pageNum, int pageSize);
	
	void addTbBrand(TbBrand tbBrand);
	
	TbBrand findOneTbBrandById(long tbBrandId);
	
	void updateTbBrand(TbBrand tbBrand);
	
	void deleteTbBrandByIds(long[] ids);
	
	PageResults findPaging(TbBrand tbBrand,int pageNum, int pageSize);
	
	List<Map> selectOptionList();

}

File: pinyougou-manager-web/src/main/webapp/js/service/brandService.js
/*品牌服务*/
app.service("brandService",function($http){
	this.findAll=function(){
		return $http.get("../brand/findAll.do");
	}
	
	this.deleteSelection=function(selectIds){
		return $http.get("../brand/deleteTbBrandByIds.do?ids="+selectIds);
	}
	
	this.addTbBrand=function(entity){
		return $http.post("../brand/addTbBrand.do",entity);
	}
	
	this.updateTbBrand=function(entity){
		return $http.post("../brand/updateTbBrand.do",entity);
	}
	
	this.findOneTbBrandById=function(tbBrandId){
		return $http.get("../brand/findOneTbBrandById.do?tbBrandId="+tbBrandId);
	}
	
	this.findPaging=function(pageNum,pageSize){
		return $http.get("../brand/findPaging.do?pageNum="+pageNum+"&pageSize="+pageSize);
	}
	
	this.search=function(pageNum,pageSize,searchEntity){
		return $http.post("../brand/searchPaging.do?pageNum="+pageNum+"&pageSize="+pageSize,searchEntity);
	}
	
	//品牌列表
	this.selectOptionList=function(){
		return $http.get('../brand/selectOptionList.do');
	}
});